renderdoc/driver/metal/metal_device.h

```diff
@@ -80,7 +94,12 @@ class WrappedMTLDevice : public WrappedMTLObject
     TypeEnum = eResDevice
   };
 
+  static uint64_t nextDrawableTLSSlot;
```

Comment:
Was not sure what naming convention to follow here?
could use "s_" to signify static or perhaps "g_" to signify it is a global static

Reply:
I don't tend to have a naming scheme for those, so this is fine as is unless you want to prefix it.